00-programming-fundamentals/github-and-the-command-line.md

```diff
@@ -0,0 +1,105 @@
+# GitHub and the Command Line
+
+So far, we've only used git locally to keep track of files on our computer. However, much of the power of git is in moving files from place to place, and enabling collaboration between people. So let's learn about how to do some of that!
+
+You might want to bookmark this page.
+
+## Learning Goals
+
+At the end of this lesson you should be able to...
+
+* Pull the contents of a GitHub repository to your computer
+* Push your changes back to GitHub
+
+## Cloning a GitHub Repository
+
+This is the procedure you'll follow at the beginning of almost every project you work on.
+
+1. Fork the repository under your own GitHub account
+  ![Git Fork Step](./images/git-fork-step.png)
+1. Check the address bar and repo title, and verify you're now on your fork
+  ![Your Fork](./images/your-fork.png)
+1. In the middle left, just above the list of files, click the green `Clone or Download` button
+  ![Clone or Download](./images/git-clone-button.png)
+1. Copy the URI that pops up (or click the `Copy to clipboard` button)
```

Comment:
For C7 I think we used the git keychain manager and folks used the HTTPS links instead of the SSH links. Does that sound familiar @CheezItMan @PilgrimMemoirs ? I think there is additional setup involved in this that we might need...